recipe/meta.yaml

```diff
@@ -28,6 +28,7 @@ requirements:
     - {{ compiler('cxx') }}
     - cmake >=3.15
     - make            # [unix]
```

Comment:
You can remove make here if you're switching to ninja.
```suggestion
```

Reply:
I left it because of the examples. But we can switch them to ninja, too.